### PR TITLE
Image drag&drop support

### PR DESCRIPTION
**Pull request by javiermarinros**
Image drag and drop support using HTML5 FileReader API, with custom styles when the editor is hovered and CSS animation when the image is loading. Tested on lastest Chrome and Firefox

### CODE REVIEW

File: src/js/medium-editor.js [hunk 1]
```diff
@@ -1029,6 +1031,64 @@ if (typeof module === 'object') {
...
+                var files;
+                e.preventDefault();
+                e.stopPropagation();
+                files = Array.prototype.slice.call(e.dataTransfer.files, 0);
+                files.some(function (file) {
+                    if (file.type.match("image")) {
+                        var fileReader = new FileReader();
+                        fileReader.readAsDataURL(file);
+
+                        var id = 'medium-img-' + Math.random();
```
Comment (nmielnik) [inline src/js/medium-editor.js]: Perhaps `Date.getTime()` could be a better uniqueId generator?

**Comment by nmielnik** [comment]
Looks like the build is failing, looking at the code I'm guessing it's some jslint issues.  Could you look into fixing those?

File: src/js/medium-editor.js (hunk 1 above)
Comment (javiermarinros) [inline src/js/medium-editor.js]: If you drag and drop two files at a time, Date.getTime() would have the same value

**Comment by javiermarinros** [comment]
Build fixed :)

**Comment by daviferreira** [comment]
:+1:

**Pull request merged**